client/src/pages/Login.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import Joi from 'joi';
+
+import axios from 'axios';
+import {
+  Avatar,
+  Button,
+  CssBaseline,
+  TextField,
+  Link,
+  Grid,
+  Box,
+  Typography,
+  Container,
+  FormControlLabel,
+  Checkbox,
+} from '@mui/material';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+
+const Copyright = (props: any) => {
+  return (
+    <Typography
+      variant="body2"
+      color="text.secondary"
+      align="center"
+      {...props}
+    >
+      {'Copyright © '}
+      <Link color="inherit" href="https://mui.com/">
+        Your Website
+      </Link>{' '}
+      {new Date().getFullYear()}.
+    </Typography>
+  );
+};
+
+const serverUrl = import.meta.env.VITE_APP_SERVER_URL;
+
+const Login = () => {
+  const [errors, setErrors] = React.useState({});
+
+  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
+    event.preventDefault();
+    const data = new FormData(event.currentTarget);
+
+    const schema = Joi.object({
+      username: Joi.string().min(3).max(30).required(),
+      password: Joi.string()
+        .min(8)
+        .max(30)
+        .required()
```

Comment:
Don't add any extra validation conditions to the backend schemas, the exact same conditions apply here.

Reply:
alsoo doneeee 👌